src/grdlandmask.c

```diff
@@ -384,7 +386,7 @@ EXTERN_MSC int GMT_grdlandmask (void *V_API, int mode, void *args) {
 	if (Ctrl->E.linetrace)
 		X = gmt_M_memory (GMT, X, nx_alloc, struct GRDLANDMASK_BINCROSS);
 
-	nx1 = Grid->header->n_columns - 1;	ny1 = Grid->header->n_rows - 1;
+	nx1 = (int)Grid->header->n_columns - 1;	ny1 = (int)Grid->header->n_rows - 1;
```

Comment:
Why are the changes for grdlandmask `int` rather than `openmp_int`?

Reply:
I think I tried and got failed tests. It seems col_min could be negative and needs to be set:

```
col_min = irint (ceil (lon_w * HH->r_inc[GMT_X] - Grid->header->xy_off));
if (col_min < 0) col_min = 0;
```

and that would fail as _openmp_int_ on macs and linux where it would be unsigned and a huge positive value.

Reply:
But I see elsewhere in that module I use MAX (0,....) to avoid this, so I will try first.

Reply:
OK, I'll rerun the tests on windows to see if this fixes the test failures.

Reply:
Thanks, I do see there is potential for negative values and hence I think it is safest to leave as ints.

Reply:
After the latest commit I get the same failures in master and this branch on windows.